mmengine/optim/optimizer/optimizer_wrapper.py

```diff
@@ -161,6 +163,16 @@ def __init__(self,
         # the loss factor will always be the same as `_accumulative_counts`.
         self._remainder_counts = -1
 
+        if len(optimizer.param_groups) > 1:
+            self.base_param_settings = {
+                'params': torch.tensor([0.0], dtype=torch.float),
+                'is_state_tracker': True
```

Comment:
Do we still need the `is_state_tracker` field?

Reply:
no, I think we no longer need this anymore, remove it.